pages/destinations/[id].js

```diff
@@ -20,6 +21,19 @@ export default function Destinations() {
   const [destinations, setDestinations] = useLocalStorage("destinations", dummyDestinations);
   const [modal, setModal] = useState({ visible: false, name: "", id: "" });
 
+  const calculateTotalDuration = () => {
+    const minDate = Math.min.apply(
+      Math,
+      destinations.map((destination) => destination.startDate)
+    );
+    const maxDate = Math.max.apply(
+      Math,
+      destinations.map((destination) => destination.endDate)
+    );
+    const dayDifference = (maxDate - minDate) / 86400 + 1;
```

Comment:
Is this what the user expects? If they travel from Saturday 4pm till Sunday 5pm they would see 2 days travel time.